### ISSUE TITLE
EISDIR: illegal operation on a directory after updating to 0.802/8.2,0

### ISSUE THREAD

**Issue by qdouble** [enten/udk#14]
When I build the project using the latest of angular, devkit and cli I get this error:

```
An error occurred during write index:
Error: EISDIR: illegal operation on a directory, open '/home/qdouble/playground/angular-universal/dist/app/browser'
    at Object.openSync (fs.js:443:3)
    at Object.writeFileSync (fs.js:1194:35)
    at Observable.rxjs_1.Observable.obs [as _subscribe] (/home/qdouble/playground/angular-universal/node_modules/@angular-devkit/core/node/host.js:176:20)
    at Observable._trySubscribe (/home/qdouble/playground/angular-universal/node_modules/@angular-devkit/core/node_modules/rxjs/internal/Observable.js:44:25)
    at Observable.subscribe (/home/qdouble/playground/angular-universal/node_modules/@angular-devkit/core/node_modules/rxjs/internal/Observable.js:30:22)
    at /home/qdouble/playground/angular-universal/node_modules/@angular-devkit/build-angular/node_modules/rxjs/internal/util/subscribeToObservable.js:10:20
    at Object.subscribeToResult (/home/qdouble/playground/angular-universal/node_modules/@angular-devkit/build-angular/node_modules/rxjs/internal/util/subscribeToResult.js:10:45)
    at SwitchMapSubscriber._innerSub (/home/qdouble/playground/angular-universal/node_modules/@angular-devkit/build-angular/node_modules/rxjs/internal/operators/switchMap.js:65:54)
    at SwitchMapSubscriber._next (/home/qdouble/playground/angular-universal/node_modules/@angular-devkit/build-angular/node_modules/rxjs/internal/operators/switchMap.js:55:14)
    at SwitchMapSubscriber.Subscriber.next (/home/qdouble/playground/angular-universal/node_modules/@angular-devkit/build-angular/node_modules/rxjs/internal/Subscriber.js:66:18)
    at SafeSubscriber.__tryOrUnsub (/home/qdouble/playground/angular-universal/node_modules/@angular-devkit/core/node_modules/rxjs/internal/Subscriber.js:207:16)
    at SafeSubscriber.next (/home/qdouble/playground/angular-universal/node_modules/@angular-devkit/core/node_modules/rxjs/internal/Subscriber.js:145:22)
    at Subscriber._next (/home/qdouble/playground/angular-universal/node_modules/@angular-devkit/core/node_modules/rxjs/internal/Subscriber.js:89:26)
    at Subscriber.next (/home/qdouble/playground/angular-universal/node_modules/@angular-devkit/core/node_modules/rxjs/internal/Subscriber.js:66:18)
    at SafeSubscriber.__tryOrUnsub (/home/qdouble/playground/angular-universal/node_modules/@angular-devkit/build-angular/node_modules/rxjs/internal/Subscriber.js:207:16)
    at SafeSubscriber.next (/home/qdouble/playground/angular-universal/node_modules/@angular-devkit/build-angular/node_modules/rxjs/internal/Subscriber.js:145:22)

```

It does it on my main project and I tried it on your angular-universal starter above. If I downgrade the packages to 8.1.3 it builds just fine.

**Comment by enten** [comment]
The issue is related to change which allow control of index output path ([8e97df38](https://github.com/angular/angular-cli/commit/8e97df38889b201aebf361ce7e8e07673411cb31)).

I published udk v1.1.7 to fix the issue.

_Can you try to update udk@1.1.7 and tell us if the issue is fixed?_

**Comment by qdouble** [comment]
Yes, it's working fine now. Hopefully, they will stop messing up the exports on every other release, lol. Thanks again!
